frontend/www/js/omegaup/components/problem/creator/cases/CaseInput.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="mt-3">
+    <b-form-group
+      :description="T.problemCreatorCaseGroupNameHelper"
+      :label="T.problemCreatorCaseName"
+      label-for="case-name"
+      class="mb-4"
+    >
+      <b-form-input v-model="caseName" required autocomplete="off" />
+    </b-form-group>
+    <b-form-group :label="T.problemCreatorGroupName" label-for="case-group">
+      <b-form-select v-model="caseGroup" :options="options" />
+    </b-form-group>
+
+    <b-form-group
+      v-show="!casePointsDefined"
+      :label="T.problemCreatorPoints"
+      label-for="case-points"
+    >
+      <b-form-input
+        v-model="casePoints"
+        type="number"
+        number
+        min="0"
+        max="100"
+      />
+    </b-form-group>
+    <b-form-group
+      :label="T.problemCreatorAutomaticPoints"
+      :description="
+        casePointsDefined ? T.problemCreatorAutomaticPointsHelper : ''
+      "
```

Comment:
```suggestion
      :description="T.problemCreatorAutomaticPointsHelper"
```

mejor que esto se muestre siempre. si no "puntaje automático" puede que no quede claro para todos.